modules/assim.batch/R/pda.bayesian.tools.R

```diff
@@ -128,8 +131,26 @@ pda.bayesian.tools <- function(settings, params.id = NULL, param.names = NULL, p
     ## Read model outputs
     model.out <- pda.get.model.output(settings, run.id, bety, inputs)
     
-    ## calculate and return likelihood
-    pda.calc.llik(settings, con, model.out, run.id, inputs, llik.fn)
+    # handle bias parameters if multiplicative Gaussian is listed in the likelihoods
+    if(any(unlist(any.mgauss) == "multipGauss")) {
+      isbias <- which(unlist(any.mgauss) == "multipGauss")
```

Comment:
Down the road, make sure to generalize this so that you can handle likelihoods with extra parameters automatically, rather than having to hack special cases in everywhere.